src/main/java/gov/usgs/wma/waterdata/groundwater/BuildRdbFile.java

```diff
@@ -75,25 +75,40 @@ protected ResultObject processRequest(String locationFolder) {
 		ResultObject result = new ResultObject();
 
 		List<String> states = locationFolderUtil.toStates(locationFolder);
+		String mess = "";
 
 		String suffix = locationFolderUtil.filenameDecorator(locationFolder);
-		if (StringUtils.isEmpty(suffix)) {
-			throw new RuntimeException("Given location folder has no state entry: " + locationFolder);
+		if (!StringUtils.hasText(suffix)) {
+			mess = "Given location folder has no state entry: " + locationFolder;
+			throw new RuntimeException(mess);
 		}
 		String filename = s3BucketUtil.createFilename(suffix);
+		String details = String.format(" [LocationFolder '%s', States: %s, S3file=%s]", locationFolder,
+			states.toString(), filename);
 
 		try (S3Bucket s3bucket = s3BucketUtil.openS3(filename)) {
 
 			Writer writer = s3bucket.getWriter();
 			RdbWriter rdbWriter = createRdbWriter(writer).writeHeader();
 			dao.sendDiscreteGroundWater(states, rdbWriter, aqDao.getParameters());
 
+			// Currently, an empty rdb file is a business rule violation as it
+			// would delete all the sites in NWISWEB for a given location (IOW-728).
+			if (rdbWriter.getDataRowCount() == 0) {
+				mess = "empty RDB file created.";
+				throw new RuntimeException(mess);
+			}
+
 			s3bucket.sendS3();
 
 			result.setCount( (int)rdbWriter.getDataRowCount() );
 			result.setMessage("Count is rows written to file: " + s3bucket.getKeyName());
+			mess = String.format("INFO: RDB file created, %d rows %s",
+					rdbWriter.getDataRowCount(), details);
+			System.out.println(mess);
```

Comment:
We want to use system.out.println() and not a logger?

Reply:
Using a logger is a todo. It will still go to cloud watch.